docs/topics/read-standard-input.md

```diff
@@ -0,0 +1,68 @@
+[//]: # (title: Read standard input)
+
+Use the `readln()` function to read data from the standard input. It reads the whole line as a string:
+
+```kotlin
+// Reads and stores the user input in a variable. For example: Hi there!
+val myInput = readln()
+
+println(myInput)
+// Hi there!
+
+// Reads and prints the user input without storing it in a variable. For example: Hi, Kotlin!
+println(readln())
+// Hi, Kotlin!
+```
+
+To work with data types other than strings, you can convert the input using conversion functions like `toInt()`, `toLong()`, `toDouble()`, `toFloat()`, and `toBoolean()` with dot syntax.
+It is possible to read multiple inputs of different data types and store each input in a variable:
+
+```kotlin
+// Converts the input from a string to an integer value. For example: 12
+val myNumber = readln().toInt()
+print(myNumber)
+// 12
+
+// Converts the input from a string to a long value. For example: 10000000000
+val myLong = readln().toLong()
+print(myLong)
+// 10000000000
+
+// Converts the input from a string to a double value. For example: 345 
+val myDouble = readln().toDouble()
+print(myDouble)
+// 345.0
+
+// Converts the input from a string to a float value. For example: 879
+val myFloat = readln().toFloat()
```

Comment:
Not sure it's worth to show conversions other than `toInt` and `toDouble`.

What we can show instead is how to read several numbers separated by a delimiter, e.g. a space - this is a very common use case in solving competition programming tasks.

e.g. 
```
val numbers = readln().split(' ').map { it.toInt() }
```
```

Reply:
I added this example and also left toBoolean() to show an example different from numbers.